docs/install.md

```diff
@@ -434,9 +439,179 @@ Configuration is relatively straightforward. As an exmaple, a minimal configurat
     server.yml: |
         # Template: https://github.com/binwiederhier/ntfy/blob/main/server/server.yml
         base-url: https://ntfy.sh
-    ```
+  ```
 
-=== "from-file"
-    ```bash
+
+from-file
+=== 
+
+  ```bash
     kubectl create configmap ntfy --from-file=server.yml 
-    ```
+  ```
+
+Kustomize-based deployment with persistent storage and Traefik ingress
```

Comment:
1. The section should just be called "Kustomize". The sidebar titles should be short
2. The whole section is a little long and has many rough edges. Please make sure it looks similar to the other install instructions when you generate the docs (mkdocs serve). I can probably help with this, but right now the PR is too rough to even do that.

Reply:
Section is cleaned up, installation steps and some references are added.